metrics/stackdriver_exporter.go

```diff
@@ -88,6 +88,10 @@ var (
 	// metricToResourceLabels provides a quick lookup from a custom metric name to the set of tags
 	// which should be promoted to Stackdriver Resource labels via opencensus resources.
 	metricToResourceLabels = map[string]*resourceTemplate{}
+
+	// A variable for testing to reduce the size (number of metrics) buffered before
+	// Stackdriver will send a bundled metric report. Only applies if non-zero.
+	TestOverrideBundleCount = 0
```

Comment:
why is this a public variable?

Reply:
It seemed like e2e tests in Serving or Eventing might want to be able to disable the batching for the same reasons that the metrics tests would want to do so.

I'd like to add tests for important exports to the Serving and Eventing repos once these tests are no longer flaky.

Reply:
Sure then let's specify that, since just looking at the comment I see no reason why it should be public.
Also let's do this in a linter happy format `TestOverrideBundleCount is a variable...`